canonical-websites/www.ubuntu.com
# Flattened ua-dell-tc

## Done

- Flattened legal data-privacy section moving it to a stand-alone file
- Updated links in the doc
- Updated the [copy doc](https://docs.google.com/document/d/1MUA0idRzWvFHlus_GoAuIo62gr03V0d4X13p8A3thVU/edit#heading=h.fxe82p9nruos)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- look at the files in the /legal directory and compare to this [spreadsheet](https://docs.google.com/spreadsheets/d/12kuuGgwRqN3k7wlRc4QKIuZifjvxbynizO_LEjKjBq4/edit#gid=0)
* test the link [/legal/ua-dell-tc](http://0.0.0.0:8001/legal/ua-dell-tc) works

## Issue / Card

Fixes #4199
